### ISSUE TITLE
Player sometimes gets stuck in the lobby

### ISSUE THREAD

**Issue by Techwizz-somboo** [zibetnu/haunting-hijinx#7]
It seems the player might occasionally get stuck in the lobby when a round starts. Seems to fix itself after the player dies, maybe?

**Comment by zibetnu** [comment]
Unfortunately, there's nothing I can do until the bug can be reproduced. It has never happened before or since the playtest where you ran into it, and there is no connection between a player dying and switching from the lobby to the level.

**Comment by Techwizz-somboo** [comment]
I'm guessing it was a dropped packet, also happened to one of my friends during our playtest. I wouldn't make it a priority to figure this one out though as it's infrequent.

**Comment by zibetnu** [comment]
There should also be log files in either `%APPDATA%\Godot\app_userdata\Haunting Hijinx` or `~/.local/share/godot/app_userdata/Haunting Hijinx`. Getting those from the person that encountered this issue in the recent playtest could help me figure out what went wrong.

**Comment by Techwizz-somboo** [comment]
Here's the log! I don't know the timestamp, but I could go through and try to find it if you believe it would be helpful.

[godot.log](https://github.com/user-attachments/files/17770002/godot.log)

**Comment by zibetnu** [comment]
Yes, a timestamp and any additional context would be helpful. The log file unfortunately doesn't have any clues for what happened.
